### PR TITLE
Read conntrack connection statistics

### PR DESCRIPTION
**Pull request by lookuptable**
This PR allows populating per-connection packet and byte counts to
ConntrackFlow object when nf_conntrack_acct is enabled.

### CODE REVIEW

**Comment by lookuptable** [comment]
@aboch @vishvananda

File: conntrack_linux.go [hunk 1]
```diff
@@ -152,10 +154,11 @@ type ConntrackFlow struct {
 func (s *ConntrackFlow) String() string {
 	// conntrack cmd output:
 	// udp      17 src=127.0.0.1 dst=127.0.0.1 sport=4001 dport=1234 [UNREPLIED] src=127.0.0.1 dst=127.0.0.1 sport=1234 dport=4001 mark=0
```
Comment (fcrisciani) [inline conntrack_linux.go]: please update the comment

File: conntrack_linux.go [hunk 2]
```diff
@@ -220,6 +223,24 @@ func parseBERaw16(r *bytes.Reader, v *uint16) {
 	binary.Read(r, binary.BigEndian, v)
 }
 
+func parseBERaw64(r *bytes.Reader, v *uint64) {
+	binary.Read(r, binary.BigEndian, v)
+}
+
+func parseByteAndPacketCounters(r *bytes.Reader) (bytes, packets uint64) {
+	for _ = range []int{1, 2} {
```
Comment (fcrisciani) [inline conntrack_linux.go]: `for i:=0; i < 2; i++`, creating a slice for a for loop of 2 is an overkill :)

**Comment by aboch** [comment]
Thanks @lookuptable please squash your commits. In future you may want to just amend your last commit  when addressing comments.

**Comment by lookuptable** [comment]
Thanks @aboch. Commits have been squashed.

**Comment by aboch** [comment]
LGTM

**Pull request merged**